packages/babel-parser/src/plugins/typescript/index.js

```diff
@@ -1466,6 +1469,36 @@ export default (superClass: Class<Parser>): Class<Parser> =>
       );
     }
 
+    tsTryParseTypeArguments(): ?N.TsTypeParameterInstantiation {
+      if (!this.isRelational("<")) {
+        return undefined;
+      }
+      const state = this.state.clone();
+      const node = this.startNode();
+      const params = this.tsInType(() =>
+        // Temporarily remove a JSX parsing context, which makes us scan different tokens.
+        this.tsInNoContext(() => {
+          if (this.eatRelational("<")) {
```

Comment:
Nit: we have already checked (at line 1473) that this is `<`

```suggestion
          this.next(); // <
```

Reply:
done